src/main/native/cpp/jni/CameraServerJNI.cpp

```diff
@@ -371,8 +364,9 @@ JNIEXPORT void JNICALL Java_edu_wpi_cscore_CameraServerJNI_setStringProperty
  * Method:    getEnumPropertyChoices
  * Signature: (I)[Ljava/lang/String;
  */
-JNIEXPORT jobjectArray JNICALL Java_edu_wpi_cscore_CameraServerJNI_getEnumPropertyChoices
-  (JNIEnv *env, jclass, jint property)
+JNIEXPORT jobjectArray JNICALL
+Java_edu_wpi_cscore_CameraServerJNI_getEnumPropertyChoices
```

Comment:
Wrapped

Reply:
allwpilib isn't consistent on this. What rules should we follow for formatting JNI signatures?

Reply:
I like keeping the same format as javah generates for easier diff'ing for changes.  Not that we couldn't write a tool that does that for you and deals with the line wrap...